src/app/components/media/CreateMedia/CreateMediaButton.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { createFragmentContainer, commitMutation, graphql } from 'react-relay/compat';
+import { browserHistory } from 'react-router';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Dialog from '@material-ui/core/Dialog';
+import { FlashMessageSetterContext } from '../../FlashMessage';
+import ButtonMain from '../../cds/buttons-checkboxes-chips/ButtonMain';
+import GenericUnknownErrorMessage from '../../GenericUnknownErrorMessage';
+import CreateMediaInput from '../CreateMediaInput';
+import { getErrorMessageForRelayModernProblem } from '../../../helpers';
+import AddIcon from '../../../icons/add.svg';
+import IconClose from '../../../icons/clear.svg';
+import styles from '../../../styles/css/dialog.module.css';
+import MediasLoading from '../MediasLoading';
+
+const CreateMediaButton = ({
+  environment,
+  team,
+}) => {
+  const [open, setOpen] = React.useState(false);
+  const [saving, setSaving] = React.useState(false);
+
+  const setFlashMessage = React.useContext(FlashMessageSetterContext);
+
+  const createMediaMutation = graphql`
+    mutation CreateMediaButtonCreateMutation($input: CreateProjectMediaInput!) {
+      createProjectMedia(input: $input) {
+        project_media {
+          dbid
+        }
+      }
+    }
+  `;
+
+  const handleError = (err) => {
+    const errorMessage = getErrorMessageForRelayModernProblem(err) || <GenericUnknownErrorMessage />;
+    setSaving(false);
+    setFlashMessage(errorMessage, 'error');
+  };
+
+  const navigateToMedia = (mediaId) => {
+    browserHistory.push(`/${team.slug}/media/${mediaId}`);
+  };
+
+  const handleSubmit = (value) => {
+    if (!value) {
+      return;
+    }
+
+    setSaving(true);
+
+    const uploadables = {};
+    if (value.file) {
+      uploadables.file = value.file;
+    }
+
+    commitMutation(environment, {
+      mutation: createMediaMutation,
+      variables: {
+        input: {
+          media_type: value.mediaType,
+          set_claim_description: value.claimDescription,
+          url: value.url,
+          quote: value.quote,
+        },
+      },
+      uploadables,
+      onCompleted: (response, err) => {
+        if (err) {
+          handleError(err);
+        } else {
+          setSaving(false);
+          setOpen(false);
+          navigateToMedia(response.createProjectMedia.project_media.dbid);
+        }
+      },
+    });
+  };
+
+  return (
+    <>
+      <ButtonMain
+        buttonProps={{
+          id: 'new-article-menu__open-button',
```

Comment:
please change the id prefix to `create-media-button__`

Reply:
Updated!